### PR TITLE
Redesigns Android Location Markers

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-06-26 at 12 52 09 PM

### What does this PR do, or why is it needed?

Makes the Android location finder pretty like iOS.

### What design trade-offs/decisions were made?

The `Locations` code definitely needs to be cleaned up but I restrained myself and only refactored what was necessary for this fix. I took out all code pertaining to the style animation because it wasn't working anyway and I'm not sure what the design requirements are. It doesn't look bad the way it is.

### How do I test this PR?

Use the location finder on iOS and Android.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [#808]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
Found the culprit, it doesn't seem that you can use absolute positions with Android custom markers

https://github.com/ApollosProject/apollos-prototype/blob/67a6e6a3733d3eb4b415ba86b870260f94073e85/packages/apolloschurchapp/src/user-settings/Locations/Marker.js#L30

**Comment by vinnyjth** [comment]
@redreceipt Code of this PR looks fantastic. You've done a ton of great cleanup work here. I'll test it when I get a chance and hit the approve button, assuming the functionality is the same + the enhanced android markers.

File: packages/apolloschurchapp/src/user-settings/Locations/index.js [hunk 1]
```diff
@@ -46,7 +40,9 @@ class Location extends PureComponent {
       latitude: 39.809734,
       longitude: -98.555618,
       latitudeDelta: 100,
-      longitudeDelta: 100 * ASPECT_RATIO,
+      longitudeDelta:
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/user-settings/Locations/index.js]: @redreceipt Can you explain this new math to me?

Comment (redreceipt) [reply packages/apolloschurchapp/src/user-settings/Locations/index.js]: I don't what `longitudeDelta` does exactly, I simply just removed some unnecessary global variables. `ASPECT_RATIO` was being defined above but I took that out.

**Comment by redreceipt** [comment]
very sneaky...I see those commits

File: packages/apolloschurchapp/src/user-settings/Locations/index.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apolloschurchapp/src/user-settings/Locations/index.js]: Gotcha. You actually are doing slightly different math here (look at the parens). I might default this to 100 and move on :)

Comment (redreceipt) [reply packages/apolloschurchapp/src/user-settings/Locations/index.js]: Oh. Prettier did that. 100*(a/b) is the same as (100*a)/b

**Pull request merged**